src/redux/actions.js

```diff
@@ -0,0 +1,14 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const fetchRandomGreeting = createAsyncThunk('greeting/fetchRandomGreeting', async () => {
+  try {
+    const response = await axios.get('http://localhost:3000/random_greeting');
+    return response.data.greeting;
+  } catch (error) {
+    console.error('Error fetching greeting:', error);
+    throw error;
+  }
```

Comment:
- [ ] Please kindly take note that it's not a good practise to have `console.log` in our codebase, it can make your app to be less secure. ℹ️  Kindly remove the `console.log` statement.

Reply:
well resolved! ✔️